pkg/operator/provisioner/provisioner.go

```diff
@@ -23,23 +23,21 @@ import (
 
 	"github.com/coreos/pkg/capnslog"
 	"github.com/kubernetes-incubator/external-storage/lib/controller"
+	"github.com/rook/rook/pkg/agent/flexvolume"
 	ceph "github.com/rook/rook/pkg/ceph/client"
-	cephmon "github.com/rook/rook/pkg/ceph/mon"
 	"github.com/rook/rook/pkg/clusterd"
-	"github.com/rook/rook/pkg/operator/k8sutil"
-	"github.com/rook/rook/pkg/operator/mon"
+	"github.com/rook/rook/pkg/operator/cluster"
 	"k8s.io/api/core/v1"
 	metav1 "k8s.io/apimachinery/pkg/apis/meta/v1"
-	"k8s.io/apimachinery/pkg/util/uuid"
 )
 
 const (
-	imageNameMaxLen = 100 // image name should be under 100 chars to support kernels older than 4.7
```

Comment:
yes.. it gives an error if its longer than 100. Not sure if its documented somewhere. But we dont have to worry about this since the plugin names the image. So it will not be longer than 100